openedx/features/smart_referral/api_views.py

```diff
@@ -23,6 +23,8 @@ def post(self, request):
         user = request.user
         user_contacts = request.data.get('contacts_list', [])
 
+        user_contacts = {contact['contact_email']: contact for contact in user_contacts}.values()
```

Comment:
How about adding a comment that this is to remove duplications. otherwise someone might be looking it the code that why are we doing this?